CommunityToolkit.Mvvm.SourceGenerators/ComponentModel/ObservableValidatorValidateAllPropertiesGenerator.cs

```diff
@@ -39,6 +40,8 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
             .Select(static (item, _) => Execute.GetInfo(item))
             .WithComparer(ValidationInfo.Comparer.Default);
 
+        context.FilterWithLanguageVersion(ref validationInfo, LanguageVersion.CSharp8, UnsupportedCSharpLanguageVersionError);
```

Comment:
The ovservable validator doesn't use C# 8 features, no? This seems unnecessary (it's not free).

Reply:
No, it isn't. As I said, this adds too much overhead. Besides, I already plan to move this out of the generator.
I'd really like for this PR to only fix that one thing so it's nice and simple and we can merge it quickly.